StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -18,9 +18,47 @@ limitations under the License.
 #include <memory>
 #include "StorageUtility.h"
 
+#include "Device.h"
+#include "DeviceDataConverter.h"
+
 namespace vtStor
 {
 
+eErrorCode GetDevicePath(String& Path, std::shared_ptr<cDeviceInterface> Device)
```

Comment:
This code is Windows specific, and should be encapsulated in the Windows cDeviceHandle class